addons/binding/org.openhab.binding.elerotransmitterstick/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="elerotransmitterstick" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="elerostick">
+		<label>Elero Transmitter Stick</label>
+		<description>Thing for an Elero Transmitter Stick</description>
+
+		<config-description>
+			<parameter name="portName" type="text" required="true">
+				<label>Port Name</label>
+				<description>The name of the port to which the Elero Transmitter Stick is connected.</description>
+			</parameter>
+			<parameter name="updateInterval" type="integer" min="5" max="3600" unit="s" step="5">
+				<label>Update Interval</label>
+				<description>The number of seconds to wait before requesting the status of the elero channels from the Elero Transmitter Stick.</description>
+				<default>30</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="elerochannel">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="elerostick" />
+		</supported-bridge-type-refs>
+
+		<label>Elero Channel</label>
+		<description>One of the 15 elero channels available on an Elero Transmitter Stick.</description>
+
+        <channels>
+            <channel id="control" typeId="control" />
+            <channel id="status" typeId="status" />
+        </channels>
+
+        <config-description>
+            <parameter name="channelId" type="integer" min="1" max="15" step="1" readOnly="true">
+                <label>Channel ID</label>
+                <description>The id of this channel.</description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+	</thing-type>
+
+	<thing-type id="elerogroup">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="elerostick" />
+		</supported-bridge-type-refs>
+
+		<label>Elero Channel Group</label>
+		<description>A group of one or more channels. Can be used to send the same command to different channels simultaneously.</description>
+
+		<channels>
+			<channel id="control" typeId="control" />
+			<channel id="status" typeId="status" />
+		</channels>
+
+		<config-description>
+			<parameter name="channelids" type="text" required="true">
+				<label>Channel IDs</label>
+				<description>Comma separated list of channel ids.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Control Channel Type -->
+	<channel-type id="control">
+		<item-type>Rollershutter</item-type>
```

Comment:
Just one more idea here: With statically linking this channel to you things you restrict yourself to only support RollerShutters. But in https://github.com/openhab/openhab2-addons/pull/2659#discussion_r140762747 you say that the stick also support "radio switching outlets". We do support [extensible thing types](https://github.com/eclipse/smarthome/pull/4301) where you can add specifically defined channels to your thing once you have created the thing. I think your generic channel things would be a good use-case, because you can turn thm into RollerShutters, Switches etc. based on which channel you add.

You can find an example in the PR for our "magic testing binding" https://github.com/eclipse/smarthome/pull/4372/files

Reply:
This is something that somebody else can contribute who owns the hardware and can test this.